plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -175,7 +204,8 @@ def main():
         policy_name=dict(required=True, type='str'),
         duration=dict(required=True, type='str'),
         replication=dict(required=True, type='int'),
-        default=dict(default=False, type='bool')
+        default=dict(default=False, type='bool'),
+        shard_group_duration=dict(required=False, type='str')
```

Comment:
```suggestion
        shard_group_duration=dict(required=False, type='str'),
```
To reduce the diff of the next addition.